frontend/src/data/utils/Constants.ts

```diff
@@ -22,5 +22,83 @@ export const UNKNOWN_RACE = "Unknown race";
 export const UNKNOWN_HL = "Unknown Hispanic or Latino";
 export const UNKNOWN_ETHNICITY = "Unknown ethnicity";
 
+export const SEX_GROUPS = [MALE, FEMALE, UNKNOWN, ALL] as const;
```

Comment:
We haven't had to define demographic groups on the frontened before and I'm curious what the gain is to adding that extra barrier?

Reply:
It's basically just creating the typescript restricted options to prevent errors in coding going forward; similar to how we define all of the `MetricId`s:
```
export type MetricId =
  | "acs_vaccine_population_pct"
  | "brfss_population_pct"
...
  ```
  
We still rely on the actual property name coming from the backend data for frontend presentation, but just confirm that the property is in our expected range of properties. Typescript gets complied away so it's mostly just preventing us from trying to assign the wrong string to these variables. Rather than define them as string unions like they do for `MetricId`, I'm defining the arrays first so we can use them for iterations / logic checks, and then generating the types from the arrays. 

I think they just didn't get a chance to strongly type everything... the code is littered with "TODO" comments to strongly type everything just to prevent headaches when trying to decipher some of the variable names. 

We can definitely leave it as strings but it seems to defeat the purpose of typescript. Is there a concern with strongly typing them?